src/auth/constants.js

```diff
@@ -2,11 +2,13 @@
 const none = -1;
 const standard = 1;
 const admin = 2;
+const superAdmin = 3;
```

Comment:
```suggestion
const standard = 0;
const admin = 1;
const superAdmin = 2;
```

This can match the backend: 
https://github.com/Code-4-Community/speak-for-the-trees-backend/pull/65#discussion_r525645175